src/views/ConfigurationJoystickView.vue

```diff
@@ -31,6 +31,16 @@
             button controls, as whel as set axis limits.
           </p>
         </div>
+        <p
+          v-if="!m2rSupportsExtendedManualControl || !ardupilotSupportsExtendedManualControl"
+          class="px-5 py-3 m-1 font-bold text-center border rounded-md text-[#856404] bg-yellow-lighten-4 max-w-[80%]"
+        >
+          It seems like you're running versions of Mavlink2Rest (BlueOS) and/or ArduPilot that do not support the
+          extended MAVLink MANUAL_CONTROL message. We strong suggest upgrading both so you can have support for
+          additional buttons and axes on the joystick. This is specially important if you still use other ground control
+          stations, like QGC, as the extended buttons prevent clashes between both softwares. We recommend minimum
+          versions of 1.2.0 for BlueOS and 4.1.2 for ArduPilot.
```

Comment:
```suggestion
          It seems like you're running versions of Mavlink2Rest (BlueOS) and/or ArduPilot that do not support the
          extended MAVLink MANUAL_CONTROL message. We strongly suggest upgrading both so you can have support for
          additional buttons and axes on the joystick. This is especially important if you sometimes use other control station
          software, like QGroundControl, as Cockpit can preferentially use the extended buttons to reduce configuration clashes. 
          We recommend using BlueOS &ge; 1.2.0, and &ge; version 4.1.2 for ArduPilot-based autopilot firmware.
```
Not sure if the "ArduPilot" bit should specifically be ArduSub, or what our preference is there.

Reply:
Yeah, had the same question. If we wanted to be super specific, we should check if it's running ArduSub, or it's another vehicle type, and act accordingly.